scene/gui/check_box.cpp

```diff
@@ -34,9 +34,7 @@
 
 Size2 CheckBox::get_icon_size() const {
 	Ref<Texture2D> checked = Control::get_theme_icon(SNAME("checked"));
-	Ref<Texture2D> checked_disabled = Control::get_theme_icon(SNAME("checked_disabled"));
 	Ref<Texture2D> unchecked = Control::get_theme_icon(SNAME("unchecked"));
-	Ref<Texture2D> unchecked_disabled = Control::get_theme_icon(SNAME("unchecked_disabled"));
```

Comment:
Instead of removing these lines, it's better to handle them properly below.

Reply:
Sure, I fixed it. Since there are so many of them, I changed it to iterate, but is this okay for null checking?
```cpp
Size2 CheckBox::get_icon_size() const {
	Vector<Ref<Texture2D>> icons;
	icons.push_back(Control::get_theme_icon(SNAME("checked")));
	icons.push_back(Control::get_theme_icon(SNAME("unchecked")));
	icons.push_back(Control::get_theme_icon(SNAME("radio_checked")));
	icons.push_back(Control::get_theme_icon(SNAME("radio_unchecked")));
	icons.push_back(Control::get_theme_icon(SNAME("checked_disabled")));
	icons.push_back(Control::get_theme_icon(SNAME("unchecked_disabled")));
	icons.push_back(Control::get_theme_icon(SNAME("radio_checked_disabled")));
	icons.push_back(Control::get_theme_icon(SNAME("radio_unchecked_disabled")));

	Size2 tex_size = Size2(0, 0);
	for (int i = 0; i < icons.size(); ++i) {
		if (icons[i].is_null()) {
			continue;
		}
		tex_size = Size2(MAX(tex_size.width, icons[i]->get_width()), MAX(tex_size.height, icons[i]->get_height()));
	}
	return tex_size;
}
```

Reply:
Yeah it's ok. But I'm concerned with performance of this approach. AFAIK pushing to Vector will re-allocate memory and `get_icon_size()` is a method that might be called relatively often. It would be better if you resized the Vector beforehand and assigned the icons for each index. Or you could use LocalVector, but I'm not sure how it works exactly.

Reply:
You have a point. It will look redundant, but I reverted to the original style for performance reasons.